packages/forms-web-app/src/dynamic-forms/question.js

```diff
@@ -318,9 +318,12 @@ class Question {
 	 * }>}
 	 */
 	formatAnswerForSummary(sectionSegment, journey, answer, capitals = true) {
+		const isAnswerProvided = answer !== null && answer !== undefined && answer !== '';
 		const formattedAnswer = capitals
 			? capitalize(answer ?? this.NOT_STARTED)
-			: answer || this.NOT_STARTED;
+			: isAnswerProvided
+			? answer
+			: this.NOT_STARTED;
```

Comment:
Both this and the getAction function were treating 0 as a falsy value. 

0 was being saved in the db and the viewModel/ journeyResponse but the client side was seeing it as falsy so was leaving the 'Not started' text on the task list page and the action button as 'Answer'